capstone/src/main/java/com/google/sps/servlets/getNYTDataServlet.java

```diff
@@ -37,19 +37,19 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     URL url = new URL(link);
     HttpURLConnection url_connect = (HttpURLConnection) url.openConnection();
     url_connect.setRequestMethod("GET");
-	int code = url_connect.getResponseCode();
+    int code = url_connect.getResponseCode();
     List<String> county_data = new ArrayList<String>();
     if(code == HttpURLConnection.HTTP_OK){ 
-	  BufferedReader reader = new BufferedReader(new InputStreamReader(url_connect.getInputStream()));
+      BufferedReader reader = new BufferedReader(new InputStreamReader(url_connect.getInputStream()));
       String input = reader.readLine();
       while(input != null) {
         input = reader.readLine();
         county_data.add(input);
-	  }
-	  reader.close();
-	}else{
+      }
+      reader.close();
```

Comment:
It would probably be best to do all of this in a try-catch-finally block. Then you won't need to throw IOException in the function signature; you can guarantee the resource won't be left unclosed if the application crashes; and you can handle the exception as you wish.

try 

Create resource that needs to be closed (reader in this case)
Any actual IO (readLine in this case)

catch IOException

handle the exception

finally

close the resource